server/package.json

```diff
@@ -32,7 +32,8 @@
     "web-push": "^3.4.4"
   },
   "scripts": {
-    "start": "nodemon --exec \"ts-node\" app.ts",
+    "start": "ts-node app.ts",
```

Comment:
In production environments I think `tsc` and `node` is preferred over `ts-node` because of performance reasons. If sticking to `ts-node` it's better to use `ts-node --transpile-only` to skip type processing.

Reply:
hmm makes sense... i think we should stick to tsc and node then to make sure the types are checked

Reply:
Yeah feel free to check out my branch where I built with `tsc` and ran `app.js`.